docs/release-notes.md

```diff
@@ -1,3 +1,17 @@
+## 0.14.2
+
+February 2, 2021
+
+### Changed
+
+* Fixed `ServerErrorMiddleware` compatibility with Python 3.9.1/3.8.7 when debug mode is enabled -
```

Comment:
I'm not a changelog pro but I'd put these 2 items under Fixed since the functionality does not change

Reply:
Yeah I had a similar thought 😅 we've only used these subheadings recently and was trying to be consistent with how it's been done in the past. All of these fall under "Fixed", I think, though.